### ISSUE TITLE
Orders panel does not show orders for marketplace shop

### ISSUE THREAD

**Issue by zenweasel** [reactioncommerce/reaction#2823]
### Expected behavior
Shop owners should expect to see orders with their products on them

### Actual Behavior
Displays "No Orders Found"

### Steps to Reproduce the Behavior
1. Create a shop
1. Create a product in that shop
1. Place an order with items from that shop and another shop
1. As the shop admin go to the orders panel
1. Observe "No Orders Found"

### Versions
```
Node: 7.10.0
NPM: 5.3.0
Meteor Node: 4.8.4
Meteor NPM: 4.6.1
Reaction CLI: 0.11.0
Reaction: 1.5.0
Reaction branch: marketplace
Docker: 17.06.2-ce

```

**Comment by spencern** [comment]
Removing the `v1.5.0` label as this doesn't really affect single shop installs. It remains critical for multi-shop installs.

**Comment by spencern** [comment]
@zenweasel I see you self-assigned 2 weeks ago, any update on this?

**Comment by zenweasel** [comment]
This was closed by #2855
